src/features/pastelStatistics/components/PrevButton/PrevButton.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import routes from '../../../../legacy/constants/routes.json'
+import styles from './PrevButton.module.css'
+
+type TPrevButton = {
+  color: string
+}
+
+const PrevButton: React.FC<TPrevButton> = ({ color }) => {
```

Comment:
Can we use JSX.Element as return type of this component instead of using React.FC?

Reply:
Updated